DualView/src/main/java/com/microsoft/device/display/samples/dualview/ui/view/MapView.kt

```diff
@@ -90,26 +85,31 @@ fun MapTopBar(isDualScreen: Boolean, customViewSize: Boolean = false) {
 }
 
 @Composable
-fun MapView(selectedIndex: Int, viewSize: Int? = null) {
+fun MapView(selectedIndex: Int) {
     var selectedMapId = R.drawable.unselected_map
     var selectedTitleId = R.string.map_description
     if (selectedIndex > nonSelection) {
         selectedMapId = restaurants[selectedIndex].mapImageResourceId
         selectedTitleId = restaurants[selectedIndex].description
     }
-    val viewSizeDp = with(LocalDensity.current) { viewSize?.toDp() }
-    val modifier = if (viewSizeDp == null) Modifier.clipToBounds() else Modifier.requiredSize(viewSizeDp)
-
-    Box(modifier = modifier.testTag(stringResource(R.string.map_image))) {
-        ScalableImageView(imageId = selectedMapId, selectedTitleId)
+    Box(
+        modifier = Modifier
+            .clipToBounds()
+            .testTag(
+                stringResource(R.string.map_image)
+            )
+    ) {
+        ScalableImageView(
+            imageId = selectedMapId, selectedTitleId
+        )
     }
 }
 
 @Preview
 @Composable
 fun MapViewScreenshotPreview() {
     DualViewAppTheme {
-        MapView(7, 150)
+        MapView(7)
     }
 }
```

Comment:
Sorry I didn't catch this before, it must be left over from when we added the tests originally! If you could remove this that would be great